src/utils.ts

```diff
@@ -17,18 +17,16 @@
 
 import * as vscode from 'vscode';
 import * as extension from './extension';
+import * as extensionFunctions from './extensionFunctions';
 import * as fs from 'fs';
 import { ViewColumn } from 'vscode';
 import * as path from 'path';
 
-export const DIDACT_DEFAULT_URL : string = 'didact.defaultUrl';
-export const DIDACT_REGISTERED_SETTING : string = 'didact.registered';
-export const DIDACT_NOTIFICATION_SETTING : string = 'didact.disableNotifications';
-export const DIDACT_COLUMN_SETTING : string = 'didact.lastColumnUsed';
-export const DIDACT_OPEN_AT_STARTUP : string = 'didact.openDefaultTutorialAtStartup';
-
-// stashed extension context
-let context : vscode.ExtensionContext;
+export const DIDACT_DEFAULT_URL = 'didact.defaultUrl';
+export const DIDACT_REGISTERED_SETTING = 'didact.registered';
+export const DIDACT_NOTIFICATION_SETTING = 'didact.disableNotifications';
+export const DIDACT_COLUMN_SETTING = 'didact.lastColumnUsed';
+export const DIDACT_OPEN_AT_STARTUP = 'didact.openDefaultTutorialAtStartup';
```

Comment:
why the type declaration is removed?

Reply:
Because ESLint told me to remove it as an initializing with a string literal already makes the type definition obsolete.

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-inferrable-types.md